core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -165,7 +189,8 @@ object ConsumerPerformance {
 
         if (currentTimeMillis - lastReportTime >= config.reportingInterval) {
           if (config.showDetailedStats)
-            printProgressMessage(0, bytesRead, lastBytesRead, messagesRead, lastMessagesRead, lastReportTime, currentTimeMillis, config.dateFormat)
+            printProgressMessage(0, bytesRead, lastBytesRead, messagesRead, lastMessagesRead,
+              lastReportTime, currentTimeMillis, config.dateFormat, Some(testStartTime), Some(joinTime.get))
```

Comment:
1. There seems to be no reason for `testStartTime` to be an `Option`. 
2. Why do we call `get` on `joinTime` and rewrap it instead of just passing it through?